Assets/StreamingAssets/Quests/10C00Y00.txt

```diff
@@ -72,6 +72,7 @@ Message:  1013
 Meridia is a Daedra Regent and a collector of human specimens.
 <--->
 Meridia is a kind of acting Daedra Prince, and ruthless in her acquisitiveness.
+--unused string
```

Comment:
Jay, if I stop comments being processed in text blocks as discussed, then this kind of thing will become part of the text block. A text block only ends once there's an empty line.

Reply:
True, I forgot that'll be changed soon. Thanks for the catch. IIRC that's the only one but I'll take another look.